stubs/PyMySQL/pymysql/connections.pyi

```diff
@@ -103,6 +103,7 @@ class Connection(Generic[_C]):
         ssl_cert=None,
         ssl_disabled=None,
         ssl_key=None,
+        ssl_key_password: str | bytes | bytearray | None = None,
```

Comment:
This appears to map to `ssl._PasswordType` in our stubs, which is `Callable[[], str | bytes | bytearray] | str | bytes | bytearray`.